content/docs/studio/user-guide/model-registry/what-is-a-model-registry.md

```diff
@@ -34,31 +71,33 @@ following interfaces:
   [model details page](/doc/studio/user-guide/model-registry/view-models#model-details-page).
 - The experiment tables for your projects also have
   [`model` columns](/doc/studio/user-guide/model-registry/view-models#model-columns-in-the-projects-experiment-table)
-  that show the different models in that project’s Git repository.
+  that show the different models in that project’s Git repository. -->
 
-<admon type="tip">
+<!-- <admon type="tip"> -->
 
 You can take a look at Iterative's [public model registry] (read only) to get a
 feel for what's possible.
 
-</admon>
+<!-- </admon> -->
 
-You can perform the following actions to the models in your model registry:
+<!-- You can perform the following actions to the models in your model registry:
 
 - [Add models to the registry](/doc/studio/user-guide/model-registry/add-a-model)
 - [Download models from the registry](/doc/studio/user-guide/model-registry/download-models)
 - [Register new versions of the models](/doc/studio/user-guide/model-registry/register-version)
 - [Assign stages to model versions](/doc/studio/user-guide/model-registry/assign-stage)
```

Comment:
I would keep these short links to the subpages over the long workflow descriptions above.